plugins/modules/zypper_repository_info.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2024, Tobias Zeuch <tobias.zeuch@sap.com>
+#
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: zypper_repository_info
+author: "Tobias Zeuch"
+short_description: List Zypper repositories
+description:
+    - List Zypper repositories on SUSE and openSUSE.
+    - Note: for info about packages, use the packages ansible.builtin.package_facts
+extends_documentation_fragment:
+    - community.general.attributes
+attributes:
+    check_mode:
+        support: none
+    diff_mode:
+        support: none
+
+requirements:
+    - "zypper >= 1.0  # included in openSUSE >= 11.1 or SUSE Linux Enterprise Server/Desktop >= 11.0"
+    - python-xml
+'''
+
+EXAMPLES = '''
+- name: List registered repositories and store in fact repositories
```

Comment:
Strictly spoken it's not a fact, but a variable :)
```suggestion
- name: List registered repositories and store in variable repositories
```